app/models/prisoner_step.rb

```diff
@@ -30,7 +30,12 @@ def prison
 
 private
 
+  # rubocop:disable Metrics/AbcSize
   def validate_prisoner
+    # If date_of_birth of number are invalid, there's no need to call the API
+    return if errors[:date_of_birth].any?
+    return if errors[:number].any?
```

Comment:
Could this not just be `return if errors[:date_of_birth].any? || errors[:number].any?`